Update 05.proteolysis.md (#25)
jessegmeyerlab/proteomics-tutorial: content/05.proteolysis.md

Proteolysis is the defining step that differentiates bottom-up or shotgun proteomics from top-down proteomics. 
Hydrolysis of proteins is extremely important because it defines the population of potentially identifyable peptides. 
Generally peptides between a length of 7-35 amino acids are considered useful for mass spectrometry analysis.

> Peptides that are too long are difficult to identify by tandem mass spectrometry, or may be lost during sample preparation due to peptides that are too short are less likely to uniquely match to a single protein. 

Reviewer: ```suggestion
Peptides that are too long are difficult to identify by tandem mass spectrometry, or may be lost during sample preparation due to irreversible binding with solid-phase extraction sorbents.
Peptides that are too short are also not useful because they may match to many proteins during protein inference. 
```

However, alternative enzymes can facilitate _de novo_ assembly when the genomic data information is limited in the public database repositories [@pmid:31615963; @pmid:30622160; @pmid:29990557; @doi:10.1016/j.actatropica.2022.106324; @DOI:10.1021/pr400173d].
Use of multiple proteases for proteome digestion also can improve the sensitivity and accuracy of protein quantification [@PMID:30336047].

Lysyl endopeptidase (Lys-C) obtained from  _Lysobacter enzymogenesis_ is a serine protease invloved in cleaving carboxyl terminus of Lys [@PMID:6359954, @PMID:25823410].
Like trypsin, the optimum pH required for its activity is from pH 7-9. 
A major advantage of Lys-C is its resistance to denaturing agents, including 8 M urea - a chaotrope commonly used to denature proteins _prior_ to digestion [@PMID:27123950].

> rypsin is less efficient in cleaving Lysthan aginine, which could limit the quality of quantitation from tryptic peptides.

Reviewer: ```suggestion
Trypsin is less efficient at cleaving Lys than Arg, which could limit the quality of quantitation from tryptic peptides.
```

> Hence, to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used in parallel to complement tryspin digestion [@PMID:23017020].

Reviewer: ```suggestion
Hence, to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used in simultaneously with trypsin digestion [@PMID:23017020].
```

Glu-C cleaves at the C-terimus of glutamate, but also after aspartate [@PMID:4627743; @PMID:26748652].

Peptidyl-Asp metallopeptidase, commonly known as Asp-N, is a metalloprotease obtained from _Pseudomonas fragi_ [@PMID:2669754].

> Asp-N catalyzes the hydrolysis of peptide bonds N-terminal of aspartate residues. 

Reviewer: ```suggestion
Asp-N catalyzes the hydrolysis of peptide bonds at the N-terminal of aspartate residues. 
```

> The optimum activity of this enzyme is between pH 4 - 9.  

Reviewer: ```suggestion
The optimum activity of this enzyme is between pH 4 and 9.  
```

Reviewer: the automated manuscript building gets confused when sentences end in what looks like math (4-9) so I changed this to 4 and 9

As with any metalloprotease, chelators like EDTA should be avoided for digestion buffers when using Asp-N.
Studies also suggest that Asp-N cleaves at the amino terminus of glutamate when a detergent is present in the proteolysis buffer [@PMID:2669754]. 
Asp-N often leaves many missed cleavages [@PMID:27123950].

> Chymotrypsin or chymotrypsinogen A is a serine protease obtained from porcine or bovine pancreas with an optimum pH range from 7.8 - 8.0 [@PMID:3555886]. 

Reviewer: ```suggestion
Chymotrypsin or chymotrypsinogen A is a serine protease obtained from porcine or bovine pancreas with an optimum pH range from 7.8 to 8.0 [@PMID:3555886]. 
```

LysargiNase, also known as Ulilysin, is a recently discovered protease belonging to the metalloprotease family. 
It is a thermophilic protease derived from _Methanosarcina acetivorans_ that specifically cleaves at the N-terminus of Lys and Arg residues [@PMID:25419962].
Hence, it enabled discovery of C-terminal peptides that were not observed using trypsin.
In addition, it can also cleave modified amino acids such as methylated or dimethylated Arg and Lys [@PMID:25419962].

Peptidyl-Lys metalloendopeptidase, or Lys-N, is an metalloprotease obtained from _Grifola frondosa_ [@PMID:19195997]. 
It cleaves N-terminally of Lys and has an optimal activity at pH 9.0.
Unlike trypsin, Lys-N is more resistant to denaturing agents and can be heated up to 70 °C [@PMID:25823410].
Reports suggest that the peptides generated after Lys-N digestion produces more of c-type ions in a ETD-based mass spectrometer [@PMID:18425140]. 
Hence this can be used for analysing PTMs, identification of C-terminal peptides and also for de novo sequencing strategies [@PMID:18425140; @PMID:20953479].

> Pepsin A, commonly known as pepsin, is an aspartic protease obtained from bovine or porcine pancreas, and was identified and crystallized and was awarded the Nobel prize [ [@PMID:12089768];@PMID:19872561;@PMID:19872562; @PMID:17758437].

Reviewer: ```suggestion
Pepsin A, commonly known as pepsin, is an aspartic protease obtained from bovine or porcine pancreas [@PMID:12089768].
Pepsin was one of several proteins crystalized by John Northrop, who shared the 1946 Nobel prize in chemistry for this work [@PMID:19872561;@PMID:19872562; @PMID:17758437;@URL:https://www.nobelprize.org/prizes/chemistry/1946/speedread/].
```

Reviewer: wanted to emphasize the nobel prize a little

> Pepsin works at an optimum pH from 1-4 and, in a pH-depended manner, it specifically cleaves Trp, Phe, Tyr and  Leu [@PMID:25823410]. 

Reviewer: ```suggestion
Pepsin works at an optimum pH from 1-4 and specifically cleaves Trp, Phe, Tyr and  Leu [@PMID:25823410]. 
```

Since it possess high enzyme activity and broad specificity at lower pH, it is preferred over other proteases for MS-based disulphide mapping [@PMID:12476442; @PMID:24980484].
Pepsin is also used extensively for structural mass spectrometry studies with hydrogen-deuterium exchange (HDX) because the rate of back exchange of the amide deuteron is minimized at low pH [@DOI:10.1021/ac902477u; @DOI:10.1002/mas.21565].  
 
Proteinase K was first isolated from the mold _Tritirachium album_ Limber [@PMID:4373242]. 

> The epithet 'K'is derived from its ability to efficiently hydrolyse keratin [@PMID:4373242].

Reviewer: ```suggestion
The epithet 'K' is derived from its ability to efficiently hydrolyse keratin [@PMID:4373242].
```

> It is a member of the subtilisin family of proteases and is relatively unspecific with a preference for cleaving at hydrophobic and aromatic amino acid residues and show optimal activity at pH 7.5 - 12 [@DOI:https://doi.org/10.1016/B978-0-12-382219-2.00714-6]. 

Reviewer: ```suggestion
It is a member of the subtilisin family of proteases and is relatively unspecific with a preference for proteolysis at hydrophobic and aromatic amino acid residues [@DOI:https://doi.org/10.1016/B978-0-12-382219-2.00714-6]. 
The optimal enzyme activity is between pH 7.5 and 12.
```

Reviewer: sentence gets a little long
